staging/src/k8s.io/code-generator/hack/verify-codegen.sh

```diff
@@ -18,8 +18,7 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-SCRIPT_ROOT=$(dirname "${BASH_SOURCE}")/..
-SCRIPT_BASE=${SCRIPT_ROOT}/../..
```

Comment:
Is `SCRIPT_BASE` used anywhere else?

Reply:
This env is not exported, and not used in this script.
After remove this, this shell can also pass.